### ISSUE TITLE
Profile Validator page and GitHub link are broken

### ISSUE THREAD

**Issue by hybridherbst** [immersive-web/webxr-input-profiles#254]
**Describe the bug**
The Profile Validator linked from the Readme currently throws a fetch error:
https://immersive-web.github.io/webxr-input-profiles/packages/viewer/dist/index.html
![image](https://github.com/immersive-web/webxr-input-profiles/assets/2693840/28c01378-ed01-4abd-bb16-90690c07d95e)

**To Reproduce**
1. Open https://immersive-web.github.io/webxr-input-profiles/packages/viewer/dist/index.html
2. Note you can't choose profiles
3. Look at the console

**Desktop (please complete the following information):**
 - OS: Android, iOS, Windows
 - Browser Chrome, Safari

**Additional context**
The link on the repo is also broken right now:
https://immersive-web.github.io/webxr-input-profiles/
![image](https://github.com/immersive-web/webxr-input-profiles/assets/2693840/acaa234b-64fa-4d7f-897b-1a8ff98b9be0)

**Comment by De-Panther** [comment]
Seems like this commit
https://github.com/immersive-web/webxr-input-profiles/commit/ec044381753a5d00154711b91cf6f50804df6f52
Replaced `motion-controllers.module.js` with `motion-controllers.js`

Searching for `motion-controllers.module.js`
https://github.com/search?q=repo%3Aimmersive-web%2Fwebxr-input-profiles+motion-controllers.module.js&type=code
Finds instances of this in files:
```
packages/viewer/rollup.config.js
packages/viewer/src/mocks/mockGamepad.js
packages/motion-controllers/README.md
packages/viewer/src/controllerModel.js
packages/viewer/src/profileSelector.js
```

**Comment by De-Panther** [comment]
Site seems down due to `remote: Permission to immersive-web/webxr-input-profiles.git denied to github-actions[bot].`

**Comment by himorin** [comment]
fixed by #258
